microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -74,6 +74,27 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
 
 
+#### Automated Quick Fixes and issue filtering
+
+Using Quick Fixes, you can customize the error reporting of the extension to the needs of the current project.
+
+When you hover over an element that has an issue, you get a lightbulb icon indicating that there are Quick Fixes available:
+
+![An anchor element with a protocol-specific href attribute highlighted as a problem, by a wavy underline and lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
+
```

Comment:
delete the file light-bulb-light.png

Reply:
"delete the file light-bulb-light.png" - done locally in new forked branch user/mikehoffms/exten-img/cln